libfwupdplugin/fu-device.c

```diff
@@ -2531,11 +2627,14 @@ fu_device_set_custom_flags (FuDevice *self, const gchar *custom_flags)
  *
  * Since: 1.1.0
```

Comment:
shouldn't this bump up to 1.6.2 since the qualifiers are changing on the return type?

Reply:
I was hoping to ignore that given that nothing now in-tree uses it.